dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/URAPIs_ADLDAP_SSLTest.java

```diff
@@ -42,54 +45,105 @@
 @RunWith(FATRunner.class)
 @Mode(TestMode.FULL)
 public class URAPIs_ADLDAP_SSLTest {
-    private static LibertyServer server = LibertyServerFactory.getLibertyServer("com.ibm.ws.security.wim.adapter.ldap.fat.ad.ssl");
+    private static LibertyServer libertyServer = LibertyServerFactory.getLibertyServer("com.ibm.ws.security.wim.adapter.ldap.fat.ad.ssl");
```

Comment:
Can we name this back to server? This rename made a bunch of changes that are just churn.